src/Microsoft.ApiDesignGuidelines.Analyzers/Core/NonConstantFieldsShouldNotBeVisible.cs

```diff
@@ -24,15 +25,25 @@ public abstract class NonConstantFieldsShouldNotBeVisibleAnalyzer : DiagnosticAn
                                                                              s_localizableMessage,
                                                                              DiagnosticCategory.Usage,
                                                                              DiagnosticSeverity.Warning,
-                                                                             isEnabledByDefault: false,
+                                                                             isEnabledByDefault: true,
                                                                              description: s_localizableDescription,
-                                                                             helpLinkUri: null,     // TODO: add MSDN url
+                                                                             helpLinkUri: "https://msdn.microsoft.com/en-us/library/ms182353.aspx",
                                                                              customTags: WellKnownDiagnosticTags.Telemetry);
 
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(Rule);
 
         public override void Initialize(AnalysisContext analysisContext)
         {
+            analysisContext.RegisterSymbolAction(obj =>
```

Comment:
These two rules are similar, but I feel like it would be confusing to reason about what each rule is if they were combined. As it is, you can clearly see what each rule is doing. I think it would be better to keep them isolated, especially as you pointed out some specific cases of the of the analyzer I didn't include. That said, I can combine them if you'd prefer that